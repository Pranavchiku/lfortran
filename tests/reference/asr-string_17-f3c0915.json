{
    "basename": "asr-string_17-f3c0915",
    "cmd": "lfortran --indent --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/string_17.f90",
    "infile_hash": "67fb24db231ad8457efddf67c0f212da514b98c084beddec67834b77",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-string_17-f3c0915.stdout",
    "stdout_hash": "ba6a02b60f985e2154c2b6d7c7bbe0f881dd61e0b9f753176e3d9bfb",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}