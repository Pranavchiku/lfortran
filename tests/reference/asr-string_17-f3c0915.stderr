warning: Implied save feature
 --> tests/../integration_tests/string_17.f90:6:9
  |
6 |         character(len=:), allocatable :: raw
  |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/string_17.f90:16:13
   |
16 |             type(string_type), intent(in) :: string
   |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/string_17.f90:17:13
   |
17 |             integer :: length
   |             ^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/string_17.f90:28:13
   |
28 |             type(string_type), intent(in) :: string
   |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/string_17.f90:29:13
   |
29 |             character(len=len(string)) :: maybe_string
   |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables
